kolibri/core/assets/src/views/exercise-attempts/answer-icon.vue

```diff
@@ -39,7 +39,7 @@
     transition: transform $core-time ease-in
 
   .yes
-    fill: $core-action-normal
+    fill: $core-status-correct
 
   .yes.yay
     transform: scale(1.5)
```

Comment:
White we are here, let's remove this unnecessary sizing that is done to the svgs

Reply:
Or make it consistent for all SVGs

![image](https://user-images.githubusercontent.com/7193975/32971769-a4032c2e-cba3-11e7-9f05-3ca6ad914eb6.png)

Reply:
that's intentional - it's supposed to be larger. Might be broken - I'll look into it

Reply:
Well in that case, don't forget about the light bulb.

Reply:
lightbulbs count as wrong, so they were 'no'

Reply:
Got it.